builders/build/runtime-upgrades.md

```diff
@@ -0,0 +1,76 @@
+|  1901   | [2317683](https://moonbeam.subscan.io/block/2317683){target=_blank} | [2911863](https://moonriver.subscan.io/block/2911863){target=_blank} | [3073562](https://moonbase.subscan.io/block/3073562){target=_blank} |
+|  2000   | [2673234](https://moonbeam.subscan.io/block/2673234){target=_blank} | [3202604](https://moonriver.subscan.io/block/3202604){target=_blank} | [3310369](https://moonbase.subscan.io/block/3310369){target=_blank} |
+|  2100   | [3011798](https://moonbeam.subscan.io/block/3011798){target=_blank} | [3588831](https://moonriver.subscan.io/block/3588831){target=_blank} | [3609708](https://moonbase.subscan.io/block/3609708){target=_blank} |
+|  2201   | [3290853](https://moonbeam.subscan.io/block/3290853){target=_blank} | [3858885](https://moonriver.subscan.io/block/3858885){target=_blank} | [3842850](https://moonbase.subscan.io/block/3842850){target=_blank} |
+|  2301   |                                  -                                  |                                  -                                   | [4172407](https://moonbase.subscan.io/block/4172407){target=_blank} |
+|  2302   | [3456477](https://moonbeam.subscan.io/block/3456477){target=_blank} | [4133065](https://moonriver.subscan.io/block/4133065){target=_blank} | [4193323](https://moonbase.subscan.io/block/4193323){target=_blank} |
+|  2401   |                                  -                                  | [4668844](https://moonriver.subscan.io/block/4668844){target=_blank} | [4591616](https://moonbase.subscan.io/block/4591616){target=_blank} |
+|  2402   |                                  -                                  |                                  -                                   | [4772817](https://moonbase.subscan.io/block/4772817){target=_blank} |
+|  2403   | [4163078](https://moonbeam.subscan.io/block/4163078){target=_blank} | [4770488](https://moonriver.subscan.io/block/4770488){target=_blank} | [4804425](https://moonbase.subscan.io/block/4804425){target=_blank} |
+|  2500   |                                  -                                  | [5175574](https://moonriver.subscan.io/block/5175574){target=_blank} | [5053547](https://moonbase.subscan.io/block/5053547){target=_blank} |
+|  2501   | [4543267](https://moonbeam.subscan.io/block/4543267){target=_blank} | [5211264](https://moonriver.subscan.io/block/5211264){target=_blank} | [5194594](https://moonbase.subscan.io/block/5194594){target=_blank} |
+|  2601   |                                  -                                  |                                  -                                   | [5474345](https://moonbase.subscan.io/block/5474345){target=_blank} |
+|  2602   | [4977160](https://moonbeam.subscan.io/block/4977160){target=_blank} | [5638536](https://moonriver.subscan.io/block/5638536){target=_blank} | [5576588](https://moonbase.subscan.io/block/5576588){target=_blank} |
```

Comment:
```suggestion
| [2601](https://forum.moonbeam.network/t/runtime-rt2600-schedule/1372/5){target=\_blank}  |                                  -                                   |                                   -                                   | [5474345](https://moonbase.subscan.io/block/5474345){target=\_blank} |
| [2602](https://forum.moonbeam.network/t/runtime-rt2600-schedule/1372/13){target=\_blank} | [4977160](https://moonbeam.subscan.io/block/4977160){target=\_blank} | [5638536](https://moonriver.subscan.io/block/5638536){target=\_blank} | [5576588](https://moonbase.subscan.io/block/5576588){target=\_blank} |
| [2700](https://forum.moonbeam.network/t/runtime-rt2700-schedule/1441/3){target=\_blank}  | [5504531](https://moonbeam.subscan.io/block/5504531){target=\_blank} | [6041969](https://moonriver.subscan.io/block/6041969){target=\_blank} | [5860584](https://moonbase.subscan.io/block/5860584){target=\_blank} |
| [2801](https://forum.moonbeam.network/t/runtime-rt2801-schedule/1616/4){target=\_blank}  |                                  -                                   |                                   -                                   | [6209638](https://moonbase.subscan.io/block/6209638){target=\_blank} |
```

Reply:
There are quite a few so I made the changes in codespace.

Reply:
Ok, thanks. Can you please add the links for 2601, 2602, 2700, and 2801 to the forum, as shown in my suggestion above?